### ISSUE TITLE
alert next to Chinese jobs numbers?

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#326]
We know (I think) that our coverage of job postings is worse outside the US, yes? So it makes sense that we get these strangely low numbers for many Chinese companies, for example
![image](https://github.com/georgetown-cset/parat/assets/58824955/5c0af212-ef45-49a6-b43b-1ff9063d0d1c)
Assuming that is the explanation - @brianlove I'm wondering whether we could/should have an alert tooltip next to the job total for every non-US company with a caveat/disclaimer.

cc @rggelles

**Comment by jmelot** [comment]
@za158 what do you think about adding the alert to the company name instead? This might avoid some layout issues

**Comment by za158** [comment]
Would prefer not - people will miss it in that case. I would like it to be as unavoidable as possible when looking at the jobs columns given how bad the issue is

If this is going to seriously mess things up we can discuss other, more radical solutions (like omitting data from those columns altogether for non-US companies)

**Comment by rggelles** [comment]
So, this is slightly more complicated than just US/non-US.

Our non-US numbers, generally, might be slightly lower than the US but should still be pretty good -- for most countries.

However, in some countries, like China, LinkedIn is banned/illegal. In those countries, our numbers are varying degrees of awful (depending on what year it was banned and how much people ignore the bans and how multinational the countries in question are). So for example, LinkedIn is blocked in China, but there was a local version that was unblocked from 2021-2023, so there is data, it's just of mixed quality. It's also blocked in Russia, Cuba, Iran, and Syria, but we don't have a lot of PARAT companies from those countries.

Some quick stats on this can be found here: https://www.apollotechnical.com/linkedin-users-by-country/

Basically the conclusion here is: countries outside the US still have a very large number of LinkedIn users! But they clearly don't have a comparable number when looking on a per capita basis to the US, so this does need a caveat. But there's a different caveat needed for countries where LinkedIn is banned outright.

I think this data is still useful for non-US non-Chinese companies, it just doesn't represent exactly the same thing as it does for US companies, so it is important to be clear.

**Comment by brianlove** [comment]
The cleanest place for the tooltip to go in the table cells would be on the left side, before the total numbers.  So, the cells would go `[tooltip if applicable] [total] [rank]`.  The component for the table cell is pretty simple and that shouldn't be an issue if we're definitely wanting the notice in that column

Current layout for reference:
![image](https://github.com/georgetown-cset/parat/assets/22353962/d16d982d-b417-4df6-840c-b0cdf5157cd7)

**Comment by za158** [comment]
@brianlove  Anywhere in that cell is fine by me

@rggelles points taken. Sounds like we should do two different tooltips - one for Chinese companies and one for non-US, non-Chinese companies. cc @brianlove

**Comment by rggelles** [comment]
Two different tooltips sounds like a good solution to me!

**Comment by brianlove** [comment]
@za158 Should these tooltips still show up when the displayed value in the table is `"n/a"`?

![image](https://github.com/georgetown-cset/parat/assets/22353962/5aa3d9d6-c0f5-4178-9b4e-8ef31ceb31fa)

**Comment by brianlove** [comment]
Also, will the contents of the tooltips be the same between AI jobs and TT1 jobs, or should the columns have different tooltips?

**Comment by za158** [comment]
Let's have a _third_ tooltip for when it's n/a. That tooltip icon should be styled differently from the other two (question mark/gray vs exclamation point/orange).

Different text for each tooltip.

**Comment by brianlove** [comment]
Using an exclamation point icon for a tooltip will require https://github.com/georgetown-cset/eto-ui-components/issues/392 (intending for the upcoming sprint).

@za158 Can you clarify which color/icon pairings you want going with which of the three tooltip cases?

**Comment by za158** [comment]
n/a tooltip: gray, question mark
China tooltip: something colorful (orange or blue), exclamation point
Country other than China or US tooltip: something colorful (orange or blue), exclamation point

**Comment by brianlove** [comment]
Do you care if the colors are the same or different for the China and non-China/non-US tooltips?

**Comment by za158** [comment]
No they can be the same
---
Zachary Arnold
Analytic Lead, Emerging Technology Observatory (web <http://eto.tech> |
LinkedIn <https://www.linkedin.com/company/emerging-technology-observatory>
| Substack <http://etoblog.substack.com>)
Center for Security and Emerging Technology
***@***.***
